src/controllers/wallet/wallet.controller.js

```diff
@@ -435,11 +435,33 @@ async function createToken(req, res) {
   const { name, symbol, amount } = req.body;
   const address = req.body.address || null;
   const changeAddress = req.body.change_address || null;
+  const createMint = req.body.create_mint === undefined ? true : req.body.create_mint;
```

Comment:
I think this would be a good place to use the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing#examples). It would make the code more readable and concise.
```suggestion
  const createMint = req.body.create_mint ?? true;
```

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-headless/pull/293/commits/6bee2632a895c9caadedbabcf014a2ee4e2a9e4e